server/src/main/java/org/opensearch/tracer/DefaultTracer.java

```diff
@@ -0,0 +1,195 @@
+package org.opensearch.tracer;
```

Comment:
Can we change the package name to either tracing or instrumentation?

Reply:
done